### DISCUSSION TITLE
TYPO3 v11 Support

### DISCUSSION THREAD

**Discussion by bernd-reindl** [oliverklee/ext-seminars#1240] [General]
Is there a plan to support TYPO3 v.11?

Best regards
Bernd Reindl

**Comment by oliverklee** [comment]
@bernd-reindl Yes, I plan to add TYPO3 11LTS compatibility with version 5.1.0.

I'm currently still working on the rewrite of my onetimeaccount extension. After that I'll set up a crowdfunding campaign to fund the work. If you're interested in contributing to the campaign, please drop me an email so I can let you know when the campaign starts.

**Comment by oliverklee** [comment]
We now have a crowdfunding campaign to finance the V11 compatibility efforts:
https://github.com/oliverklee/ext-seminars/wiki/11LTS-Crowdfunding-Campaign

**Comment by oliverklee** [comment]
Now that the first few milestones of the [crowdfunding campaign](https://github.com/oliverklee/ext-seminars/wiki/11LTS-Crowdfunding-Campaign) have received sufficient backing, I have started work on them. I'll regularly send out emails to keep the backers and other interested persons up to date on the latest developments for this. If you also would like to receive these emails, please drop me an email (e.g., to the email address listed [on my GitHub profile](https://github.com/oliverklee/)) or chat me up on the TYPO3 Slack and let me know your email address.

@peerlight Interested?
